Annex60/ThermalZones/ReducedOrder/Validation/VDI6007/package.order

```diff
@@ -1,7 +1,4 @@
 TestCase1
-TestCase10
```

Comment:
Yes, but I think we then also need to rename regression test files, the links to them and the respective result files as well.